src/assets/locales/fr.json

```diff
@@ -273,6 +308,15 @@
   "more-menu.tos": "Conditions d'utilisation",
   "more-menu.unfavorite-room": "Supprimer la salle des favoris",
   "more-menu.whats-new": "Quoi de neuf",
+  "new-token-scopes.read-rooms.description": "Lire les données de la salle",
+  "new-token-scopes.write-rooms.description": "Écrire les données de la salle",
+  "new-token.back": "Retour",
+  "new-token.generate": "Générer",
+  "new-token.select-scopes-description": "Régler le niveau d'accès que ce jeton aura en choisissant à partir de la liste des portées d'accès.",
+  "new-token.select-scopes-title": "Sélectionner les portées d'accès",
+  "new-token.title": "Nouveau jeton",
+  "new-token.token-type": "Type de jeton",
+  "new-token.warning-at-least-one-scope": "Veuillez sélectionner au moins une portée d'accès.",
```

Comment:
"new-token-scopes.read-rooms.description": "Read room data",
  "new-token-scopes.write-rooms.description": "Write room data",
  "new-token.back": "Back",
  "new-token.generate": "Generate",
  "new-token.select-scopes-description": "Set the level of access this token will have by choosing from the scopes list.",
  "new-token.select-scopes-title": "Select scopes",
  "new-token.title": "New Token",
  "new-token.token-type": "Token type",
  "new-token.warning-at-least-one-scope": "Please select at least one scope.",

Reply:
```suggestion
  "new-token.warning-at-least-one-scope": "Veuillez sélectionner au moins un périmètre d'accès.",
```